### ISSUE TITLE
The homepage feels a bit "dry"

### ISSUE THREAD

**Issue by oncletom** [GFDRR/open-risk-data-dashboard#206]
The homepage has very few informations about the intent of the project.

During the UX/UI assessment, it felts it could benefit of stating clearer things related to the “there is a need to assess the state of open data in the sector, enable users to search for open data in an easier way, identify data gaps and enable Governments and data producers to see where they stand and make progress.” bit of the Terms of Reference.

- what are we going to get when we click on "Explore locations"?
- what does it mean to read "29.6% open data"? What's next?
- what is implied by "submit a dataset"? Can it be "share a link from a data provider" or "upload mine" kind of type?
- what could convey more of a sense of community on this page? (number of contributors? registered persons?)

**Comment by pzwsk** [comment]
Yes we agree, should we discuss Homepage in a dedicated session together with @gracedoherty or maybe start here with some screenshot propositions?

I think we all have some ideas.
